src/schemas/21-526EZ/schema.js

```diff
@@ -331,7 +309,34 @@ let schema = {
       required: ['servicePeriods', 'servedInCombatZone'],
       properties: {
         servicePeriods: {
-          $ref: '#/definitions/servicePeriods'
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              serviceBranch: {
+                type: 'string',
+                enum: [
+                  'Air Force',
+                  'Air Force Reserve',
+                  'Air National Guard',
+                  'Army',
+                  'Army National Guard',
+                  'Army Reserve',
+                  'Coast Guard',
+                  'Coast Guard Reserve',
+                  'Marine Corps',
+                  'Marine Corps Reserve',
+                  'National Oceanic and Atmospheric Administration',
+                  'Navy',
+                  'Navy Reserve',
+                  'Public Health Service'
+                ]
+              },
+              dateRange: {
+                $ref: '#/definitions/dateRangeAllRequired'
+              }
+            }
```

Comment:
Should we keep the `minItems`, `maxItems`, and required fields, or were these causing issues in pre-fill?

Reply:
Yes, we should definitely keep them. 🤦‍♂️